### DISCUSSION TITLE
Please help me to collect drink names!

### DISCUSSION THREAD

**Discussion by teruto725** [NAIST-SE/AttractingContributors#83] [General]
Please tell me about your favorite drink name!
https://github.com/teruto725/AttractingContributors-Drink/
* please pull request to main branch

**Comment by disk-12** [comment]
I added coke

**Reply by teruto725** [reply]
Thank you !

**Comment by yokoro13** [comment]
I added!
Please help me!
https://github.com/yokoro13/AttractingContributors-VideoGames

**Reply by teruto725** [reply]
I've already done!

**Comment by sergiodl-naist** [comment]
I added Mezcal!
Please visit and try to ask a question or recommend one vtuber on my fork!
https://github.com/sergiodl-naist/AttractingContributors-VTubers

**Comment by iikyara** [comment]
I added MITSUYA CIDER!
Please help me!
https://github.com/iikyara/AttractingContributors-Gifts

**Reply by teruto725** [reply]
Thank you! I did!

**Comment by GuangxianZhu** [comment]
Ginger Ale, I like it.

**Reply by GuangxianZhu** [reply]
https://github.com/LeeHomchi/AttractingContributors_Sports
This is link. Thank you.
Welcome to my question.

**Comment by nagata-atsuki** [comment]
added
Please help me
https://github.com/nagata-atsuki/AttractingContributors-Athlete

**Comment by veekay7** [comment]
I added C.C. Lemon.
Please help.
https://github.com/VantelinLuca/AttractingContributors-WishCountry
